app/react/src/server/framework-preset-react-dom-hack.ts

```diff
@@ -0,0 +1,22 @@
+import { readJSON } from 'fs-extra';
```

Comment:
I think you just need `JSON.parse(await readFile("filename.json", "utf8"))` ...

Add one dependence only for this feature is costly

Reply:
`fs-extra` is used in other parts of the storybook ecosystem, therefore @ndelangen decided to just use it without thinking too much about the bundle size. For sure, we can remove the fs-extra part, if JSON.parse(await readFile) doesn't introduce a different behavior. @ndelangen What do you think?

Reply:
Merging this for now to keep things moving. Feel free to address this in a followup PR. I agree we should be trimming down our dependencies wherever possible if it doesn't add complexity, but could go either way on this